lib/mm_network.pm

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/perl -w
+
+package mm_network;
+
+use base Exporter;
+use Exporter;
+
+use strict;
```

Comment:
If we ever decide to add perl-critic checks this will trigger 'code before strictures'.